code/modules/mob/living/carbon/human/say.dm

```diff
@@ -37,13 +37,27 @@
 				winset(client, "input", "text=[null]")
 
 /mob/living/carbon/human/speech_bubble_appearance()
-	if(isSynthetic())
-		var/datum/robolimb/robo = isSynthetic()
-		return robo.speech_bubble_appearance
-	else
-		if(species)
-			return species.speech_bubble_appearance
-	return "normal"
+	var/sounds_synth = FALSE
+	var/datum/robolimb/robo = isSynthetic() //Will get torso manufacturer
+	if(robo)
+		sounds_synth = looksSynthetic() //Based on lifelike robolimb vars
+
+	// Not lifelike and got manufacturer
+	if(sounds_synth)
+		return robo.speech_bubble_appearance || "synthetic"
+
+	// Not lifelike synth, might have synth voice box
+	if(!robo)

```

Comment:
That's badass, nice. Really happy to see this checks for synth voice implants now.